crypto/ffc/ffc_dh.c

```diff
@@ -148,6 +157,7 @@ int ossl_ffc_named_group_set_pqg(FFC_PARAMS *ffc, const DH_NAMED_GROUP *group)
 
     ossl_ffc_params_set0_pqg(ffc, (BIGNUM *)group->p, (BIGNUM *)group->q,
                              (BIGNUM *)group->g);
+    ffc->keylength = group->keylength;
```

Comment:
I think we should rename this function if it does more than just set p, q and g.

Reply:
I think we will never merge this PR.

Reply:
OK renamed as another fixup was needed.